api/com_res/main.py

```diff
@@ -42,84 +33,8 @@
     allow_headers=["*"],
 )
 
-app.include_router(
-    argo_router,
-    # prefix="/auth/cuahsi",
-    tags=["argo"],
-)
-
-app.include_router(
-    utilities_router,
-    tags=["utilities"],
-)
-
-app.include_router(
-    access_control_router,
-    # prefix="/auth/cuahsi",
-    tags=["minio"],
-)
-
-app.include_router(
-    storage_router,
-    # prefix="/auth/cuahsi",
-    tags=["minio"],
-)
 
 app.include_router(
-    hydroshare_router,
-    # prefix="/auth/cuahsi",
-    tags=["hydroshare"],
+    timeseries_router,
+    tags=["timeseries"],
 )
-
-app.include_router(
-    fastapi_users.get_oauth_router(
-        cuahsi_oauth_client,
-        auth_backend,
-        SECRET,
-        redirect_url=get_settings().oauth2_redirect_url,
-        associate_by_email=True,
-    ),
-    prefix="/auth/cuahsi",
-    tags=["auth"],
-)
-
-app.include_router(
-    fastapi_users.get_oauth_router(
-        cuahsi_oauth_client,
-        auth_backend,
-        SECRET,
-        redirect_url=get_settings().vite_oauth2_redirect_url,
-        associate_by_email=True,
-    ),
-    prefix="/auth/front",
-    tags=["auth"],
-)
-
-app.include_router(
-    fastapi_users.get_users_router(UserRead, UserUpdate),
-    prefix="/users",
-    tags=["users"],
-)
-
-
-@app.on_event("startup")
-async def on_startup():
-    await init_beanie(
-        database=db,
-        document_models=[
-            User,
-        ],
-    )
-    arguments = [
-        "mc",
-        "alias",
-        "set",
-        "cuahsi",
-        f"https://{get_settings().minio_api_url}",
-        get_settings().minio_access_key,
-        get_settings().minio_secret_key,
-    ]
-    try:
-        _output = subprocess.check_output(arguments)
-    except subprocess.CalledProcessError as e:
-        raise
```

Comment:
we don't think this app will ever use Minio?

Reply:
This was giving me issues because I didn't configure MinIO in my development environment. I suppose that we might use this in the future, but for now we aren't.